test/unit/dat-tcp_tests.rb

```diff
@@ -218,17 +209,11 @@ class WorkLoopWithWorkTests < ListenAndRunningTests
     end
 
     should "accept connections and add them to the worker pool" do
-      assert_includes @client, @worker_pool_spy.work_items
+      assert_includes @client, @wp_spy.work_items
     end
 
-    should "allow retrieving the client sockets file descriptors" do
-      assert_includes @client.fileno, subject.client_file_descriptors
-    end
-
-    should "client file descriptors should still be accessible after its paused" do
-      @io_select_stub.set_data_on_signal_pipe
-      @server.pause true
-      assert_includes @client.fileno, subject.client_file_descriptors
```

Comment:
@kellyredding - I originally removed it because the client file descriptors are known no matter what the state of the server, but its not a bad test to have for any future changes so I restored it. Its a good check to ensure we still have access to the client file descriptors after we paused, otherwise Sanford can't hot restart correctly.